### PR TITLE
POL-414 Adding AzureMCA and Google to Superseded and Cheaper Regions

### PR DESCRIPTION
**Pull request by lawrethao**
### Description

Adds support for Google on superseded_instances.pt and Google/AzureMCA on cheaper_regions.pt

### Issues Resolved

https://jira.flexera.com/browse/POL-414

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Review by rshade — DISMISSED** [review]
LGTM

File: cost/superseded_instance/superseded_instance.pt [hunk 1]
```diff
@@ -229,6 +237,11 @@ script "js_new_costs_request", type: "javascript" do
         {"dimension":"service", "type":"equal", value:"Microsoft.Compute"},
         {"dimension":"resource_type","type":"substring","substring":"Virtual Machines"}
       ]},
+      {"type" : "and", "expressions" : [
+        {"dimension":"vendor","type":"equal","value":"Google"},
```
Comment (cdwilhelm) [inline cost/superseded_instance/superseded_instance.pt:241]: @lawrethao   can you please make sure this is the correct dimension?  is it Google or GCP?

**Review by cdwilhelm — COMMENTED** [review]
see the one comment I made

**Review by rshade — APPROVED** [review]
LGTM

**Pull request merged**